features/_your_results/limit_tax_relief.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Limit Tax Relief
+  In order to get a clear picture of the impact on my take-home pay.
+  As an employee contributing greater than £40,000 per year to my pension
+  I want the results to reflect the fact that I will not get tax relief
+  for any contribution over £40,000 per year.
+
+  Background:
+    Given I am on the Your Details step
+
+    Scenario: Limit tax relief when above yearly figures
```

Comment:
This scenario can be explained like this `Scenario: Employee contributing more than £40,000 per year to pension`